src/storage/src/hummock/utils.rs

```diff
@@ -389,7 +389,15 @@ pub fn check_subset_preserve_order<T: Eq>(
     true
 }
 
-pub(crate) const ENABLE_SANITY_CHECK: bool = cfg!(debug_assertions);
+static SANITY_CHECK_ENABLED: AtomicBool = AtomicBool::new(cfg!(debug_assertions));
+
+pub fn set_sanity_check_enabled(enabled: bool) {
```

Comment:
Better to make it `fn disable_sanity_check()`. Otherwise, when the value of `enabled` is true, we will still do sanity check in release, though the passed value is always false.

Besides, may add a comment that this method can only be called for once.